### ISSUE TITLE
logp accuracy error.

### ISSUE THREAD

**Issue by owenustc** [isayev/ReLeaSE#27]
Hi,
    I repeat to execute your LogP module with two test.  But, in first test, I only get 0.2413 for valid SMILES  when I  used retrained generator and predictor to do reinforcement learning.   And my retrained generator also get 0.8876497315159025 for drug-like region and 0.7263 for valid SMILES before reinforcement learning.  I don't know why my generator get the low valid SMILES after Reinforcement learning.  
And in second test, I get  0.7698 for valid SMILES and 0.9664848012470771 for  drug-like region  when I used your checkpoint of generator.  Do you have some tricks for generator training?

**Comment by owenustc** [comment]
I use the default configuration of the code to run generator training.

**Comment by owenustc** [comment]
My generator got a high drug-like region, about 0.99, but very lower valid SIMILES, only 0.27.  I just to re-execute your generator re-training code.   Could you help me?

**Comment by isayev** [comment]
@owenustc just filter out invalid SMILES if you need more samples.

Also, training RL models require some experience and practice; look up for the learning rates and gradients. Better take RL class if you are not sure what to change.

**Comment by owenustc** [comment]
@isayev，In your opinion,  do you think the generator with lower valid SIMILES  is right?  I take the RL class of your code to do RL training for generator.   And do you meet this situation that the generator valid SMILES  lower than 0.3 after RL trainning?     Because I use your checkpoint of generator to do RL training and the valid SIMILES is more than 0.7,  I guess  the checkpoint training of your generator model could have some tricks.
